src/core/metrics/sources/nginx_access_log.go

```diff
@@ -56,6 +56,10 @@ var logVarMap = map[string]string{
 	"]":                         "\\]",
 }
 
+// Pattern to match all the variables that are mentioned in the access log
+// and are absent from logVarMap
+var logVarRegex = regexp.MustCompile(`\$([a-zA-Z]+[_[a-zA-Z]+]*)`)
```

Comment:
nit: the regex pattern itself doesn't exclude things from the `logVarMap`, that bit of docs might be misleading

```suggestion
// Pattern to match all the variables that are mentioned in the access log
// format
var logVarRegex = regexp.MustCompile(`\$([a-zA-Z]+[_[a-zA-Z]+]*)`)
```